packages/flutter/lib/src/foundation/assertions.dart

```diff
@@ -379,6 +379,7 @@ class FlutterErrorDetails with Diagnosticable {
   /// The [exception] must not be null; other arguments can be left to
   /// their default values. (`throw null` results in a
   /// [NullThrownError] exception.)
+  // If you add a property to this class, do not forget to update [copyWith].
```

Comment:
should we say this on every class with a copyWith?

Reply:
That's fair, I had this in here before adding copy with. Removing.